module-types/Arista/CCS-750X-48THP-LC.yaml

```diff
@@ -0,0 +1,150 @@
+---
+manufacturer: Arista
+model: CCS-750X-48THP-LC
+part_number: CCS-750X-48THP-LC
+comments: '[Arista 750X Series Datasheet](https://www.arista.com/assets/data/pdf/Datasheets/CCS-750-Series-Datasheet.pdf)'
+interfaces:
+  - name: Ethernet{module}/1
+    type: 1000base-t
+    poe_mode: pd
```

Comment:
If these ports output poe they should be `poe_mode: pse` and then the required `poe_type:` e.g `poe_type: type2-ieee802.3at`